save-core/src/commonMain/kotlin/org/cqfn/save/core/utils/TomlUtils.kt

```diff
@@ -56,45 +51,23 @@ private inline fun <reified T : PluginConfig> Path.createPluginConfig(
  * @return list of plugin configs from toml file
  * @throws PluginException in case of unknown plugin
  */
-fun createPluginConfigListFromToml(testConfigPath: Path): MutableList<PluginConfig> {
-    val configList: MutableList<PluginConfig> = mutableListOf()
-    val parsedTomlConfig = TomlParser(testConfigPath.toString()).readAndParseFile()
-    parsedTomlConfig.getRealTomlTables().forEach { tomlPluginSection ->
-
-        // adding a fake file node to restore the structure and parse only the part of the toml
-        // this is a hack for easy partial read of Toml configuration
-        val fakeFileNode = TomlFile()
-        tomlPluginSection.children.forEach {
-            fakeFileNode.appendChild(it)
-        }
-
-        val sectionName = tomlPluginSection.name.uppercase()
-        // we don't convert sectionName to enum, because we don't want to get Kotlin exception
-        val sectionPluginConfig = when (sectionName) {
-            TestConfigSections.FIX.name -> testConfigPath.createPluginConfig<FixPluginConfig>(
-                fakeFileNode,
-                sectionName
-            )
-            TestConfigSections.FIX_AND_WARN.name -> testConfigPath.createPluginConfig<FixAndWarnPluginConfig>(
-                fakeFileNode,
-                sectionName
-            )
-            TestConfigSections.WARN.name -> testConfigPath.createPluginConfig<WarnPluginConfig>(
-                fakeFileNode,
-                sectionName
-            )
-            TestConfigSections.GENERAL.name -> testConfigPath.createPluginConfig<GeneralConfig>(
-                fakeFileNode,
-                sectionName
-            )
-            else -> throw PluginException(
-                "Received unknown plugin section name in the input: [$sectionName]." +
-                        " Please check your <$testConfigPath> config"
-            )
+fun createPluginConfigListFromToml(testConfigPath: Path): List<PluginConfig> =
+    TomlParser(KtomlConf())
+        .readAndParseFile(testConfigPath.toString())
+        .getRealTomlTables()
+        .map {
+            when (TestConfigSections.valueOf(it.fullTableName.uppercase())) {
+                TestConfigSections.FIX -> testConfigPath.createPluginConfig<FixPluginConfig>(
+                    it.fullTableName
+                )
+                TestConfigSections.`FIX AND WARN` -> testConfigPath.createPluginConfig<FixAndWarnPluginConfig>(
```

Comment:
Could you please update name of this section in corresponding README of `FIX AND WARN` plugin

Reply:
How did it work before? In readme you had [fix-and-warn], in Enum you had FIX_AND_WARN

Reply:
For this moment we have no examples for it https://github.com/cqfn/save/issues/170
However there is exist unit test in `commonTest` directory for this plugin